specutils/io/default_loaders/wcs_fits.py

```diff
@@ -23,15 +26,15 @@ def identify_wcs1d_fits(origin, *args, **kwargs):
 
 
 @data_loader("wcs1d-fits", identifier=identify_wcs1d_fits,
-             dtype=Spectrum1D)
-def wcs1d_fits(file_name, spectral_axis_unit=None, **kwargs):
+             dtype=Spectrum1D, extensions=['fits'])
+def wcs1d_fits_loader(file_name, spectral_axis_unit=None, **kwargs):
     """
-       Parameters
-       ----------
-       file_name : str
+    Parameters
```

Comment:
this docstring needs an opening sentence.  Or more generally: what *is* this for?  That is, what *kind* of "WCS1D" are we talking about here?  I'm 95% sure think you mean WCS paper III spectral format, right? If so, awesome, but say that (with reference to the paper).

Reply:
This docstring still does not tell the user what this loader is for - i.e., it doesn't address the questions I asked above in this comment.

Reply:
This is strange. I _did_ update this, and you can see the change [here](https://github.com/nmearl/specutils/blob/extend-data-loaders/specutils/io/default_loaders/wcs_fits.py#L30). It doesn't seem to be showing the github diff...

Reply:
Right, I see it in the diff but what I'm saying is that it does not answer the above information. That is, "in FITS-files with defined WCS" is not sufficient because it's not clear what *kind* of WCS you mean.

That said, I don't want this to be a big blocker, so it's fine to defer this for later as long as you make an issue for it though, @nmearl .

Reply:
I'm actually unsure of what you're really getting at here. Do you mean _kind_ as in "the assumptions about the structure of the WCS in terms of columns/hdus/etc", or do you mean kind as in "it's a FITS WCS"? The latter is plainly stated by the docstring, but the former would require the user to know what it's looking for in the fits wcs.

Reply:
I meant more the former, and that's precisely the point: this docstring has to be written for a user who has a file and is saying "is the appropriate for my file?"  And there's not enough information here for the user to know that.  That is, there are fits files out in the wild that are tabular but *also* have a WCS in the header.  This is not meant for that use case, because it assumes the `flux` data are the data in the "D" of the HDU.  It also assumes that hdu 0 is the only HDU of interest, etc.

I'll make a PR against your branch to demonstrate what I mean.  But the broader point is that once we make these "public", these docstrings have to have sufficient detail that the reader can understand what kind of file is needed without looking at the code itself (which is what I had to do).

Reply:
(See nmearl/specutils#22)